wallet/chain/p/builder/builder_test.go

```diff
@@ -0,0 +1,183 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+//nolint:gosec // This file does not need cryptographically strong randomness
```

Comment:
Can this be limited to `G404`?

Reply:
Is there a way to do this without littering `// #nosec G404` throughout the file?

Reply:
[4yo open issue](https://github.com/golangci/golangci-lint/issues/1165) requesting the feature :(